articles/active-directory/manage-apps/use-scim-to-provision-users-and-groups.md

```diff
@@ -858,7 +859,7 @@ Developers using the CLI libraries provided by Microsoft for building a SCIM ser
        SystemIdentityModel.Tokens.TokenValidationParameters tokenValidationParameters =     
          new TokenValidationParameters()
          {
-           ValidAudience = "00000002-0000-0000-c000-000000000000"
+           ValidAudience = "02050662-6776-fn00-c000-0780093v0n00"
```

Comment:
The valid Audience should also be the same as the parent app id (8adf8e6e-67b2-4cf2-a259-e3dc5476c621)

Reply:
Also, v and n are not valid characters in a GUID.